<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="4" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-05-11T01:52:12 IST" name="iNetBanking" finished-at="2020-05-11T01:56:30 IST" duration-ms="257353">
    <groups>
    </groups>
    <test started-at="2020-05-11T01:52:12 IST" name="iNetBanking" finished-at="2020-05-11T01:56:30 IST" duration-ms="257353">
      <class name="com.iNetBanking.testCases.TC_LoginDDT_002">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.iNetBanking.testCases.TC_LoginDDT_002@4278a03f]" started-at="2020-05-11T01:52:51 IST" name="setUp" finished-at="2020-05-11T01:53:11 IST" duration-ms="19675" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.iNetBanking.testCases.TC_LoginDDT_002@4278a03f]" started-at="2020-05-11T01:53:26 IST" name="loginDDT" data-provider="LoginData" finished-at="2020-05-11T01:53:52 IST" duration-ms="26108" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr259451]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[YtYpUbE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.iNetBanking.testCases.TC_LoginDDT_002@4278a03f]" started-at="2020-05-11T01:53:52 IST" name="loginDDT" data-provider="LoginData" finished-at="2020-05-11T01:54:01 IST" duration-ms="9029" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[harshayk]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.iNetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.iNetBanking.testCases.TC_LoginDDT_002@4278a03f]" started-at="2020-05-11T01:54:01 IST" name="loginDDT" data-provider="LoginData" finished-at="2020-05-11T01:54:19 IST" duration-ms="17389" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr259451]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[YtYpUbE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.iNetBanking.testCases.TC_LoginDDT_002@4278a03f]" started-at="2020-05-11T01:54:19 IST" name="loginDDT" data-provider="LoginData" finished-at="2020-05-11T01:54:29 IST" duration-ms="10082" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[harshayk]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.iNetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.iNetBanking.testCases.TC_LoginDDT_002@4278a03f]" started-at="2020-05-11T01:54:29 IST" name="tearDown" finished-at="2020-05-11T01:54:30 IST" duration-ms="1565" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.iNetBanking.testCases.TC_LoginDDT_002 -->
      <class name="com.iNetBanking.testCases.TC_LoginTest_001">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.iNetBanking.testCases.TC_LoginTest_001@2f177a4b]" started-at="2020-05-11T01:52:13 IST" name="setUp" finished-at="2020-05-11T01:52:39 IST" duration-ms="25720" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginTest()[pri:0, instance:com.iNetBanking.testCases.TC_LoginTest_001@2f177a4b]" started-at="2020-05-11T01:52:39 IST" name="loginTest" finished-at="2020-05-11T01:52:49 IST" duration-ms="10884" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.iNetBanking.testCases.TC_LoginTest_001@2f177a4b]" started-at="2020-05-11T01:52:50 IST" name="tearDown" finished-at="2020-05-11T01:52:51 IST" duration-ms="1493" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.iNetBanking.testCases.TC_LoginTest_001 -->
      <class name="com.iNetBanking.testCases.TC_AddCUstomerTest_003">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.iNetBanking.testCases.TC_AddCUstomerTest_003@147ed70f]" started-at="2020-05-11T01:54:30 IST" name="setUp" finished-at="2020-05-11T01:54:52 IST" duration-ms="21346" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="addNewCustomer()[pri:0, instance:com.iNetBanking.testCases.TC_AddCUstomerTest_003@147ed70f]" started-at="2020-05-11T01:54:52 IST" name="addNewCustomer" finished-at="2020-05-11T01:56:28 IST" duration-ms="96634" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCustomer -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.iNetBanking.testCases.TC_AddCUstomerTest_003@147ed70f]" started-at="2020-05-11T01:56:28 IST" name="tearDown" finished-at="2020-05-11T01:56:30 IST" duration-ms="1469" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.iNetBanking.testCases.TC_AddCUstomerTest_003 -->
    </test> <!-- iNetBanking -->
  </suite> <!-- iNetBanking -->
</testng-results>
